src/_data/styles/stylecss-stephym9170.json

```diff
@@ -0,0 +1,6 @@
+{
+"title": "style stage",
```

Comment:
Please use a more "custom" title - sorry, I should have a guideline about it!

Also just checking that all your styles came through as you intended - perhaps you want to do a bit more to the layout or add some more color?
https://deploy-preview-164--moderncss-styles.netlify.app/styles/style-stage/